keras_nlp/tokenizers/tokenizer_test.py

```diff
@@ -57,3 +89,24 @@ def test_functional_model(self):
     def test_missing_tokenize_raises(self):
         with self.assertRaises(NotImplementedError):
             Tokenizer()(["the quick brown fox"])
+
+    def test_detokenize_to_strings_for_ragged(self):
+        input_data = ["▀▁▂▃", "samurai"]
+        tokenizer = PassThroughTokenizer()
+        tokenize_output = tokenizer.tokenize(input_data)
+        detokenize_output = tokenizer.detokenize_to_strings(tokenize_output)
+        self.assertAllEqual(detokenize_output, ["▀▁▂▃", "samurai"])
+
+    def test_detokenize_to_strings_for_dense(self):
+        input_data = ["▀▁▂▃", "samurai"]
+        tokenizer = PassThroughTokenizer()
+        tokenize_output = tokenizer.tokenize(input_data, sequence_length=5)
+        detokenize_output = tokenizer.detokenize_to_strings(tokenize_output)
```

Comment:
Same, here just pass a `tf.constant([...])`, don't bother with a sequence_length argument, and don't call tokenize in this test.